src/main/java/net/neoforged/neoforge/network/registration/NetworkRegistry.java

```diff
@@ -111,13 +105,19 @@ public class NetworkRegistry {
      * Registry of all custom payload handlers. The initial state of this map should reflect the protocols which support custom payloads.
      * TODO: Change key type to a combination of protocol + flow.
      */
-    private static final Map<ConnectionProtocol, Map<ResourceLocation, PayloadRegistration<?>>> PAYLOAD_REGISTRATIONS = ImmutableMap.of(
+    protected static final Map<ConnectionProtocol, Map<ResourceLocation, PayloadRegistration<?>>> PAYLOAD_REGISTRATIONS = ImmutableMap.of(
+            ConnectionProtocol.CONFIGURATION, new HashMap<>(),
+            ConnectionProtocol.PLAY, new HashMap<>());
+    protected static final Map<ConnectionProtocol, Map<ResourceLocation, IPayloadHandler<?>>> SERVERBOUND_HANDLERS = ImmutableMap.of(
+            ConnectionProtocol.CONFIGURATION, new HashMap<>(),
+            ConnectionProtocol.PLAY, new HashMap<>());
+    protected static final Map<ConnectionProtocol, Map<ResourceLocation, IPayloadHandler<?>>> CLIENTBOUND_HANDLERS = ImmutableMap.of(
             ConnectionProtocol.CONFIGURATION, new HashMap<>(),
             ConnectionProtocol.PLAY, new HashMap<>());
```

Comment:
Ah. that's where it is. I see we're cheating a bit making these protected to bring them in scope via extending this class.

If it works it works, I guess, lol